### PR TITLE
Added support for rules creation

### PR DESCRIPTION
**Pull request by prashanthpx**

### CODE REVIEW

File: drivers/volume/volume.go [hunk 1]
```diff
@@ -169,7 +169,7 @@ const (
 )
 
 var (
-	volDrivers = make(map[string]Driver)
+	volDrivers   = make(map[string]Driver)
```
Comment (siva-portworx) [inline drivers/volume/volume.go:172]: Why this change?

Comment (prashanthpx) [reply drivers/volume/volume.go:172]: As aprt of make, it has done formatting and aligned this.
